'mc cp' with no arguments panics when printing help

The `mc cp` with no argument panics because of some issues with printing help. `mc cp --help` also panics.

```
% mc cp
panic: template: help:62: unexpected "}" in operand

goroutine 1 [running]:
text/template.Must(...)
	text/template/helper.go:23
github.com/minio/cli.printHelpCustom(0x4a86ec0, 0xc0000ce008, 0x4994162, 0xe42, 0x49426a0, 0xc0002a0600, 0x0)
	github.com/minio/cli@v1.22.0/help.go:260 +0x514
github.com/minio/cli.ShowCommandHelp(0xc0002376c0, 0x494c37d, 0x2, 0xc0001a5568, 0x400e546)
	github.com/minio/cli@v1.22.0/help.go:203 +0x4ae
github.com/minio/cli.ShowCommandHelpAndExit(0xc0002376c0, 0x494c37d, 0x2, 0x1)
	github.com/minio/cli@v1.22.0/help.go:167 +0x3f
github.com/minio/mc/cmd.checkCopySyntax(0xc0002376c0, 0xc000283b00)
	github.com/minio/mc@/cmd/cp-url-syntax.go:29 +0x74e
github.com/minio/mc/cmd.mainCopy(0xc0002376c0, 0x0, 0x2376c0)
	github.com/minio/mc@/cmd/cp-main.go:558 +0xd6
github.com/minio/cli.HandleAction(0x482c1c0, 0x49a2e80, 0xc0002376c0, 0xc0002a2000, 0x0)
	github.com/minio/cli@v1.22.0/app.go:498 +0xc8
github.com/minio/cli.Command.Run(0x494c37d, 0x2, 0x0, 0x0, 0x0, 0x0, 0x0, 0x495511c, 0xc, 0x0, ...)
	github.com/minio/cli@v1.22.0/command.go:225 +0x991
github.com/minio/cli.(*App).Run(0xc0002361a0, 0xc0000de020, 0x2, 0x2, 0x0, 0x0)
	github.com/minio/cli@v1.22.0/app.go:261 +0x6ab
github.com/minio/mc/cmd.Main(0xc0000de020, 0x2, 0x2)
	github.com/minio/mc@/cmd/main.go:113 +0x29c
main.main()
	github.com/minio/mc@/main.go:34 +0x45
```

```
% mc --version
mc version RELEASE.2020-01-13T22-49-03Z
```

macOS Catalina (10.15.1 -- 19B88)

--- a/cmd/cp-main.go
+++ b/cmd/cp-main.go
@@ -144,7 +144,7 @@ EXAMPLES:
       {{.Prompt}} {{.HelpName}} -a myobject.txt play/mybucket
 
   16. Copy a text file to an object storage with object lock mode set to 'GOVERNANCE' with retention date.
-      {{.Prompt}} {{.HelpName} --attr "x-amz-object-lock-mode=GOVERNANCE;x-amz-object-lock-retain-until-date=2020-01-11T01:57:02Z" locked.txt play/locked-bucket/
+      {{.Prompt}} {{.HelpName}} --attr "x-amz-object-lock-mode=GOVERNANCE;x-amz-object-lock-retain-until-date=2020-01-11T01:57:02Z" locked.txt play/locked-bucket/
 `,
 }
 

